lib/Zyberspace/Telegram/Cli/RawClient.php

```diff
@@ -62,7 +62,12 @@ public function exec($command)
             if (substr($answer, 0, 7) === 'ANSWER ') {
                 $bytes = (int) substr($answer, 7);
                 if ($bytes > 0) {
```

Comment:
I will add the trim with additional requirements (it needs extra simple
care) and then send a pull request

On Tue, Jun 2, 2015, 7:28 PM Eric Enold <notifications@github.com> wrote:

> In lib/Zyberspace/Telegram/Cli/RawClient.php
> <https://github.com/zyberspace/php-telegram-cli-client/pull/6#discussion_r31541769>
> :
>
> > @@ -62,7 +62,12 @@ public function exec($command)
> >              if (substr($answer, 0, 7) === 'ANSWER ') {
> >                  $bytes = (int) substr($answer, 7);
> >                  if ($bytes > 0) {
>
> [image: Codacy]
> <https://camo.githubusercontent.com/b1b1514cd8a7d7c9c9fa32f769284569ffee2882/68747470733a2f2f7777772e636f646163792e636f6d2f76657273696f6e65642f696d616765732f66617669636f6e2e706e67>
> Issue found: Expected 1 newline after opening brace; 2 found
> <https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/699691092/issues/source?bid=2302274&fileBranchId=2378797#l64>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zyberspace/php-telegram-cli-client/pull/6/files#r31541769>
> .
>